deeppavlov/models/spelling_correction/levenshtein/levenshtein_searcher.py

```diff
@@ -211,52 +210,51 @@ def _minimal_replacement_cost(self, first, second):
         return min(removal_cost, insertion_cost)
 
 
-def _precompute_absense_costs(dictionary, removal_costs, insertion_costs, n,
-                              allow_spaces=False):
+def _precompute_absense_costs(dictionary, removal_costs, insertion_costs, n, allow_spaces=False):
     """
-    Вычисляет минимальную стоимость появления нового символа в узлах словаря
-    в соответствии со штрафами из costs
+    Calculates the minimum cost for a new character to appear in dictionary nodes
+    according to fines from costs
 
-    Аргументы:
+    Arguments:
     ---------------
-    dictionary : Trie
-        словарь, хранящийся в виде ациклического автомата
+    dictionary: Trie
+        dictionary stored as an acyclic automaton
 
-    removal_costs : dict
-        штрафы за удаление символов
+    removal_costs: dict
+        character removal fines
 
-    insertion_costs : dict
-        штрафы за вставку символов
+    insertion_costs: dict
+        character insertion fines
 
-    n : int
-        глубина ``заглядывания вперёд'' в словаре
+    n: int
+        depth of `` looking ahead '' in the dictionary
 
-    Возвращает
+    Returns
     ---------------
-    answer : list of dicts, len(answer)=len(dictionary)
-        answer[i][a][j] равно минимальному штрафу за появление символа a
-        в j-ой позиции в вершине с номером i
+    answer: list of dicts, len (answer) = len (dictionary)
+        answer [i] [a] [j] is equal to the minimum penalty for the appearance of the character a
+        at the j-th position at the vertex number i
     """
     answer = [dict() for node in dictionary.data]
     if n == 0:
         return answer
     curr_alphabet = copy.copy(dictionary.alphabet)
     if allow_spaces:
-        curr_alphabet += [' ']
+        curr_alphabet += [" "]
```

Comment:
Why the single quotes hate? 😃

Reply:
Because  the single quote is often used in English, for example "it's something". And it is result of black code formatter